packages/vite/src/node/plugins/resolve.ts

```diff
@@ -555,10 +552,21 @@ function tryCleanFsResolve(
     if (dirStat?.isDirectory()) {
       if (possibleJsToTs) {
         // try resolve .js, .mjs, .mts or .jsx import to typescript file
```

Comment:
```suggestion
        // try resolve .js, .mjs or .jsx import to typescript file
```

Looks like we can update this comment too.

Reply:
it should be .cjs, updated :+1: